packages/components/src/components/table/Table.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Checkbox, Icon, Dropdown } from '..';
+
+export type TableProps = {
+  items: any[];
+  header: any[];
+  checkbox?: boolean;
+  className?: string;
```

Comment:
You are currently not using this prop,
you can remove it as long as you don't use it